.gitignore

```diff
@@ -1,3 +1,9 @@
-.next
+# dependencies
 node_modules
 
+# build output
+.next
+
+# logs
+.DS_Store
```

Comment:
@MAPESO I wouldn't consider this as a `logs` file, maybe create another section for OS files

Reply:
good idea. I will delete `.DS_Store` :)